docs/Quickstart.md

```diff
@@ -0,0 +1,229 @@
+# Quickstart
+
+> To set up development machines manually instead of using the IoT Edge Dev Container, please see the [Manual Development Machine Setup Wiki](https://github.com/Azure/iotedgedev/wiki/manual-dev-machine-setup).
+
+This quickstart will run a container, create a solution, setup Azure resources, build and deploy modules to your device, setup and start the IoT Edge simulator, monitor messages flowing into IoT Hub, and finally deploy to the IoT Edge runtime.
+
+Using IoT Edge Dev Container is the absolute fastest way to get IoT Edge Dev Tool (aka `iotedgedev`) running on your dev machine.
+
+You can use the IoT Edge Dev Tool container to avoid having to install all the dependencies on your local dev machine. The only thing you need to install is Docker. All of the other dev dependencies are included in the container.
+
+<!-- Here's a 3 minute video walk-through of this Quickstart:
+
+[![Azure IoT Edge Dev Tool: Quickstart](assets/edgedevtoolquickstartsmall.png)](https://aka.ms/iotedgedevquickstart) -->
+
+The only thing you need to install is Docker. All of the other dev dependencies are included in the container.
+
+1. **Install [Docker CE](https://docs.docker.com/install/)**
```

Comment:
Consider using a different step numbering scheme as follows:
1-
   a.
   b.
     i -
     ii -
2-

Reply:
Unfortunately  markdown does not support letters or numbers natively, so I'd rather avoid trying to use them